ui/app/models/azure/config.js

```diff
@@ -14,7 +14,12 @@ export default class AzureConfig extends Model {
   @attr('string', { label: 'Tenant ID' }) tenantId;
   @attr('string', { label: 'Client ID' }) clientId;
   @attr('string', { sensitive: true }) clientSecret; // obfuscated, never returned by API
-  @attr('string') environment;
+
+  @attr('string', {
+    subText:
+      'This value can also be provided with the AZURE_ENVIRONMENT environment variable. If not specified, Vault will use Azure Public Cloud.',
```

Comment:
This text was in the full Azure build out designs, and it seemed helpful so I added.